src/compiler/emitter.ts

```diff
@@ -4551,10 +4572,11 @@ namespace ts {
                 }
 
                 previousSibling = child;
+                previousSiblingEmitFlags = undefined;
             }
 
             // Write a trailing comma, if requested.
-            const emitFlags = previousSibling ? getEmitFlags(previousSibling) : 0;
+            const emitFlags = previousSibling ? previousSiblingEmitFlags ?? getEmitFlags(previousSibling) : 0;

```

Comment:
isn't `previousSiblingEmitFlags` always undefined here? It looks like it's reset to undefined at the bottom of the for loop, and there aren't any breaks or continues that I see.